File: dayOne/JavaScript/LevelOne/PalindromeArray.js
function isPalindrome(array) {
    
    left_index = 0;
    right_index =  - 1;
    
    while (left_index < right_index) {
        if (array[left_index] !== array[right_index]) {
            return false;
        }    
        left_index++;
        right_index--;
    }
    return true;
    
}


const number = [45, 0, 8, 0, 45];

console.log("Array is Palindrome:", isPalindrome(number));

File: dayOne/Java/LevelOne/PalindromeArray.java
public class PalindromeArray {

//    public static void main(String[] args) {
//        int[] numbers = {45, 0, 8, 0, 45};
//               
//        System.out.println("Array is Palindrome: " + isPalindrome(numbers));
//    }
    
    
    public static boolean isPalindrome(int[] array) {
        int leftIndex = 0;
        int rightIndex = array.length - 1;
    
        while (leftIndex < rightIndex) {
            if (array[leftIndex] != array[rightIndex]) {
                return false;
            }
            leftIndex++;
            rightIndex--;
        }
            
        return true;        
    }
    
}
